src/Service/QueueVideoImports.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace Drupal\media_mpx\Service;
+
+use Drupal\Core\Logger\LoggerChannelInterface;
+use Drupal\Core\Queue\QueueFactory;
+use Drupal\media\MediaTypeInterface;
+use Drupal\media_mpx\DataObjectFactoryCreator;
+use Drupal\media_mpx\DataObjectImporter;
+use Drupal\media_mpx\Event\ImportSelectEvent;
+use Drupal\media_mpx\MpxImportTask;
+use Drupal\media_mpx\Repository\MpxMediaType;
+use Lullabot\Mpx\DataService\Fields;
+use Lullabot\Mpx\DataService\ObjectListIterator;
+use Lullabot\Mpx\DataService\ObjectListQuery;
+use Lullabot\Mpx\DataService\Range;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+/**
+ * Class QueueVideoImports.
+ *
+ * @package Drupal\media_mpx\Service
+ */
+class QueueVideoImports {
+
+  const MEDIA_MPX_IMPORT_QUEUE = 'media_mpx_importer';
```

Comment:
Should we remove the constant at `\Drupal\media_mpx\Commands\MpxImporter::MEDIA_MPX_IMPORT_QUEUE` to use this one?

Reply:
@deviantintegral I'd leave this one as is for now. Since I'll try to make the importer reuse the queue service, I should be able to get rid of the one in from `\Drupal\media_mpx\Commands\MpxImporter`.